Creating-and-Starting-Demo-Projects.md

```diff
@@ -0,0 +1,122 @@
+	<br><br>
+    <code>init</code> creates a folder with the specified name
+    (<code>demo</code> in this case) and copies the files you need
+    into it. You can give the project any name you like. This
+    documentation assumes you used <code>demo</code>.. 
+    <br><br>
+    There are optional arguments that let you specify which project
+    you want to initialize. By default, their values are set to
+    initialize a project we wrote to introduce you to Agoric smart
+    contracts. See the (<b>tyg todo</b> insert links) Agoric CLI page
+    or Creating New Contracts or Using Existing Contracts page for
+    details on these optional arguments.
+  <br><br>If this 
+  doesn't work, use <code>echo $PATH</code> to check that your Agoric
+      CLI install directory is in your <code>$PATH</code> If not, add
+      it to <code>$PATH</code></td>
+  </tr>
+  <tr>
+    <td><center>3</center></td>
+    <td>Run <code>cd demo</code></td>
+    <td>Move to the directory where your project (the demo) is
+  located.</td> 
+  </tr>
+  <tr>
+    <td><center>4</center></td>
+    <td>Run <code>agoric install</code></td>
+    <td>Install JavaScript dependencies, which may take a while..</td>
+  </tr>
+  <tr>
+    <td><center>5</center></td>
+    <td>Run <code>agoric start --reset</code></td>
+    <td>Start the Agoric VM. <code>--reset</code> discards any prior Agoric
+  state. This creates the <i>vats</i> in which smart contract code runs.
+  The VM continues to run in this shell, making it unusable for
+  running commands.</td>
+  </tr>
+  <tr>
+    <td><center>6</center></td>
+    <td>Open another shell, and run <code>cd ~/agoric/sdk/demo</code></td>
+    <td>Use the same project directory name as you used in Step 2. In
+      this example, we used <code>demo</code> but you may have used a
+      different word in Step 2.</td>
+  </tr>
+  <tr>
+    <td><center>7</center></td>
+    <td>In the new shell, run <code>agoric deploy ./contract/deploy.js ./api/deploy.js</code></td>
+    <td>Deploy the Dapp on the Agoric VM, installing its smart
+  contract and web API, as well as JavaScript code for writing and implementing
+      contracts and for the Agoric API. (<b>tyg todo</b>: Is this right?)</td>
+  </tr>
+  <tr>
+    <td><center>8</center></td>
+    <td>In the new shell, run <code>cd ui</code></td>
+    <td>Move into the sub-directory that stores the Autoswap demo's UI</td></td>
+  </tr>
+  <tr>
+    <td><center>9</center></td>
+    <td>In the new shell, run <code>yarn install</code></td>
+    <td>Install NPM dependencies</td>
+  </tr>
+  <tr>
+    <td><center>10</center></td>
+    <td>In the new shell, run <code>yarn start</code></td>
+    <td><code>start</code> is a <code>yarn</code> sub-command
+  implemented by Agoric. It lauunches the React development server.</td>
+  </tr>
+  <tr>
+    <td><center>11</center></td>
+    <td>Go to a browser and open <code>http://localhost:3000</code> to
+  see our demo Dapp (<i>Distributed Application</i>)</td>
```

Comment:
This is too many expansions of "Dapp". It's already explained earlier on this page.